test/parallel/test-http2-compat-serverresponse-write.js

```diff
@@ -71,3 +71,26 @@ const assert = require('assert');
     }));
   }));
 }
+
+{
+  const server = createServer();
+  server.listen(0, mustCall(() => {
+    const port = server.address().port;
+    const url = `http://localhost:${port}`;
+    const client = connect(url, mustCall(() => {
+      const request = client.request();
+      request.resume();
+      request.on('end', mustCall());
+      request.on('close', mustCall(() => {
+        client.close();
```

Comment:
Are both this and `client.destroy();` below needed. I would keep only one if possible.

Reply:
I think is not necessary, just pushed a new commit for that. Thanks